src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1074,6 +1074,16 @@ private DetectProperties() {
             .setGroups(DetectGroup.MAVEN, DetectGroup.SOURCE_SCAN)
             .build();
 
+    public static final BooleanProperty DETECT_MAVEN_INCLUDE_SHADED_DEPENDENCIES =
+            BooleanProperty.newBuilder("detect.maven.include.shaded.dependencies",false)
+                    .setInfo("Include Shaded Dependencies", DetectPropertyFromVersion.VERSION_9_5_0)
+                    .setHelp(
+                            "If set to true, Detect will include shaded dependencies as part of BOM.",
+                            "A shaded dependency is packaged inside the uber jar of the direct or transitive dependency referenced in the project. Detect will find the use of maven-shade-plugin from original POM file and based on that will derive information for these dependencies. This property will only be supported in build mode just like all other MAVEN properties. "
```

Comment:
Since this feeds into public docs it would be good to remove the trailing space after `.`.